Packages/com.unity.inputsystem/InputSystem/Actions/InputActionMap.cs

```diff
@@ -1984,6 +1984,7 @@ public void OnAfterDeserialize()
         {
             m_State = null;
             m_MapIndexInState = InputActionState.kInvalidIndex;
+            m_EnabledActionsCount = 0;
```

Comment:
Nice small resolution to this one :) I would recommend considering adding a unit test that captures this scenario if this wasn't previously covered in the unit tests for InputActionMap. Do you think that would be feasible?

Reply:
I added 2 test cases:
- An Actions test covering the specific scenario
- A ProjectWideActions test validating Maps/Actions cannot be added/removed while ActionMap is enabled

The only other time ActionMap's `m_State` is reset is when processing Setup changes, i.e. `OnSetupChanged()`, which is only allowed when it's already disabled. I didn't find any other test validating this specific behavior and so added the second case.

Reply:
Sounds great. Thank you.